api/src/test/java/com/alibaba/nacos/api/naming/pojo/healthcheck/HealthCheckerFactoryTest.java

```diff
@@ -48,6 +48,7 @@ public void testDeserialize() {
     
     @Test
     public void testDeserializeExtend() {
+        HealthCheckerFactory.registerSubType(TestChecker.class, TestChecker.TYPE);
```

Comment:
I think solve this problem by add @BeforeClass method. And register subType in @BeforeClass method will be better.

Reply:
Sounds good. Will update soon.

Reply:
Done